scripts/workers/code_upload_submission_worker.py

```diff
@@ -274,6 +288,241 @@ def install_gpu_drivers(api_instance):
             raise
 
 
+def create_namespace(api_instance):
+    """Function to create a namespace on AWS EKS cluster
+    Arguments:
+        api_instance {[AWS EKS API object]} -- API object for creating namespace
+    """
+    body = client.V1Namespace(
+        api_version="v1",
+        kind="Namespace",
+        metadata=client.V1ObjectMeta(
+            name="amazon-cloudwatch", labels={"name": "amazon-cloudwatch"}
+        ),
+    )
+    try:
+        api_instance.create_namespace(body)
+    except ApiException as e:
+        if e.status == 409:
+            logging.info("Namespace Already created")
+        else:
+            raise
+
+
+def create_service_account(api_instance):
+    """Function to create a service_account on AWS EKS cluster
+    Arguments:
+        api_instance {[AWS EKS API object]} -- API object for creating service_account
+    """
+    namespace = "amazon-cloudwatch"
+    body = client.V1ServiceAccount(
+        api_version="v1",
+        kind="ServiceAccount",
+        metadata=client.V1ObjectMeta(name="fluentd"),
+    )
+    try:
+        api_instance.create_namespaced_service_account(
+            namespace=namespace, body=body,
+        )
+    except ApiException as e:
+        if e.status == 409:
+            logging.info("Service Account Already created")
+        else:
+            raise
+
+
+def create_cluster_role(api_instance):
+    """Function to create a cluster_role on AWS EKS cluster
+    Arguments:
+        api_instance {[AWS EKS API object]} -- API object for creating cluster_role
+    """
+    body = client.V1ClusterRole(
+        api_version="rbac.authorization.k8s.io/v1",
+        kind="ClusterRole",
+        metadata=client.V1ObjectMeta(name="fluentd-role"),
+        rules=client.V1PolicyRule(
+            api_groups=[""],
+            resources=["namespaces", "pods", "pods/logs"],
+            verbs=["get", "list", "watch"],
+        ),
+    )
+    try:
+        api_instance.create_cluster_role(body=body)
+    except ApiException as e:
+        if e.status == 409:
+            logging.info("cluster_role Already created")
```

Comment:
Can you please be a bit more verbose here as which cluster is this and what binding is this?